data/templates/nginx/server.tpl.conf

```diff
@@ -42,7 +42,14 @@ server {
     # > openssl dhparam -out /etc/ssl/private/dh2048.pem -outform PEM -2 2048
     #ssl_dhparam /etc/ssl/private/dh2048.pem;
 
-    add_header Strict-Transport-Security "max-age=31536000;";
+    add_header Strict-Transport-Security "max-age=63072000; includeSubDomains; preload"; 
+    add_header 'Referrer-Policy' 'same-origin';
```

Comment:
Need to be tested on django apps before to apply it.

Reply:
The feature to test on apps is the login feature of apps

Reply:
Yes. for better testing I will split the PR in 2.